redoc/build.js

```diff
@@ -0,0 +1,87 @@
+const { execFile } = require('child_process')
+const { promisify } = require('util')
+const { resolve, join } = require('path')
+const cpFile = require('cp-file')
+const fs = require('fs')
+const readFile = promisify(fs.readFile)
+const writeFile = promisify(fs.writeFile)
+
+const SWAGGER_PATH = resolve(__dirname, '..', 'swagger.yml')
+const LOGO_PATH = resolve(__dirname, 'netlify-logo.png')
+const OUTPUT_DIR = resolve(__dirname, 'dist')
+const OUTPUT_PATH = resolve(OUTPUT_DIR, 'index.html')
+const OUTPUT_JS_PATH = resolve(OUTPUT_DIR, 'js')
+const OUTPUT_LOGO_PATH = resolve(OUTPUT_DIR, 'netlify-logo.png')
+
+const pExecFile = promisify(execFile)
+
+// Build API documentation single self-contained HTML file using `redoc-cli`
+const buildDocs = async function() {
+  await Promise.all([
+    redocCli(),
+    copyAssets(),
+  ])
+  // Inject analytics
+  await addAnalytics()
+}
+
+const redocCli = async function() {
+  await pExecFile('redoc-cli', [
+    `--title=${TITLE}`,
+    `--options.expandResponses=${SUCCESS_STATUS_CODES}`,
```

Comment:
Thoughts on disabling this? 

When 200 response are auto expanded by default it makes scrolling much much longer.

I tried the build with this line commented out and it was easier to quickly scroll down the page.

Reply:
Fixed in 275ecc4afc61484c1a8f3ffcf9a5fbb2a8a22125